ChangeLog.md

```diff
@@ -52,6 +52,8 @@
 
 - Redefines `Fail` as a synonym for `Throw String`. ([#247](https://github.com/fused-effects/fused-effects/pull/247))
 
+- Removes `Resumable` and its carriers, as they can be more usefully and flexibly replaced by arbitrary effects, `Lift`, and `InterpretC`. ([#269](https://github.com/fused-effects/fused-effects/pull/269))
```

Comment:
I’m honestly debating whether we need to re-home it or whether just removing it is fine.

We’ll probably want to copy it into `semantic` as a short-term migration thing, but I don’t think it really _deserves_ its own package—tho better options (e.g. coroutines) might.

Reply:
I think we should move it to a package, and when we land a better solution (e.g. coroutines) we can mark the package as deprecated. That’s the most graceful solution, in my view, given that I bragged about `Resumable` at Strange Loop and I don’t want to renege on that particular promise. What do you think?

Reply:
Yeah, that’s a good call 👍

Reply:
Yeah, that’s a good call 👍

Reply:
Gonna hold off merging this PR until we have a repo & URL for that.